### PR TITLE
fix: adapt command runner to package manager in schematics

### PR DESCRIPTION
**Pull request by cpaulve-1A**
## Proposed change

Do not force yarn on schematics as we both support npm and yarn

### CODE REVIEW

**Review by vscaiceanu-1a — DISMISSED** [review]

File: packages/@o3r/core/schematics/rule-factories/cms-adapter/index.ts [hunk 1]
```diff
@@ -97,8 +98,9 @@ export function updateCmsAdapter(options: { projectName: string | null }, rootPa
   const addExtractersScripts = (tree: Tree, _context: SchematicContext) => {
     const workspaceProject = getProjectFromTree(tree);
     const packageJson = readPackageJson(tree, workspaceProject);
+    const command = getPackageManagerRunner();
```
Comment (pginoux-1A) [inline packages/@o3r/core/schematics/rule-factories/cms-adapter/index.ts:101]: Command can be renamed to packageManagerRunner for more clarity

File: packages/@o3r/core/schematics/ng-add-create/index.ts [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import { chain, move, Rule, SchematicContext, Tree } from '@angular-devkit/schematics';
+import { getPackageManagerRunner } from '@o3r/dev-tools';
```
[the remark this answers is not in the document]
Comment (cpaulve-1A) [reply packages/@o3r/core/schematics/ng-add-create/index.ts:2]: in core or schematics?

**Review by pginoux-1A — APPROVED** [review]

File: packages/@o3r/core/schematics/ng-add-create/index.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (kpanot) [reply packages/@o3r/core/schematics/ng-add-create/index.ts:2]: Yes, maybe schematics would make more sense in the end

**Review by pginoux-1A — APPROVED** [review]

File: packages/@o3r/schematics/schematics/ng-add/index.ts [hunk 3]
```diff
@@ -16,15 +15,15 @@ import { NodePackageName } from '@angular-devkit/schematics/tasks/package-manage
 class DevInstall extends NodePackageInstallTask {
   /** @inheritDoc */
   public toConfiguration() {
-    const installOptions = getPackageManager();
+    const installOptions = process.env && process.env.npm_execpath && process.env.npm_execpath.indexOf('yarn') === -1 ? 'npm' : 'yarn';
```
Comment (kpanot) [inline packages/@o3r/schematics/schematics/ng-add/index.ts:18]: why not importing it from the `src`?

Comment (cpaulve-1A) [reply packages/@o3r/schematics/schematics/ng-add/index.ts:18]: could do now that it is in the schematics

Comment (cpaulve-1A) [reply packages/@o3r/schematics/schematics/ng-add/index.ts]: Nope, you can't refer to the source until the peerdep are resolved, hence I cannot use the one from src.

**Review by kpanot — APPROVED** [review]

**Pull request merged**